### PR TITLE
Add [junos_lag_interfaces] resource module

### PR DESCRIPTION
**Pull request by ganeshrn**
##### SUMMARY

*  Add new resource module junos_lag_interfaces.
*  Targets model https://github.com/ansible-network/resource_module_models/pull/29
*  Deprecate junos_linkagg module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

junos_lag_interfaces

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/module_utils/network/junos/argspec/lag_interfaces/lag_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        pass
+
+    argument_spec = {'config': {'elements': 'dict',
+                                'options': {'members': {'elements': 'dict',
+                                                        'options': {'link_type': {'choices': ['primary',
+                                                                                              'backup']},
+                                                                    'member': {'type': 'str'}},
+                                                        'type': 'list'},
+                                            'mode': {'choices': ['active', 'passive']},
+                                            'name': {'required': True, 'type': 'str'},
```
Comment (trishnaguha) [inline lib/ansible/module_utils/network/junos/argspec/lag_interfaces/lag_interfaces.py:46]: As discussed we are going to have `id` instead of `name`.

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

File: lib/ansible/modules/network/junos/junos_lag_interfaces.py [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+# }
+# ae0 {
+#     description "lag interface";
+# }
+# ae1 {
+#     description "lag interface 1";
+# }
+
+- name: "Delete LAG attributes of given interfaces (Note: This won't delete the interface itself)"
+  junos_lag_interfaces:
```
Comment (justjais) [inline lib/ansible/modules/network/junos/junos_lag_interfaces.py:127]: since `name` is required param, so is this example correct for delete.

Comment (justjais) [reply lib/ansible/modules/network/junos/junos_lag_interfaces.py:127]: also @ganeshrn , I checked running the operation merge with the current PR, and I am getting the following output: http://paste.openstack.org/show/754529/, where before and after is coming empty.

Comment (ganeshrn) [reply lib/ansible/modules/network/junos/junos_lag_interfaces.py:127]: Fixed

**Pull request merged**